monkey/monkey_island/cc/ui/src/components/reactive-graph/ReactiveGraph.js

```diff
@@ -12,12 +12,14 @@ class GraphWrapper extends React.Component {
     let newOptions = null;
     if(this.props.options !== undefined){
       newOptions = this.props.options;
-      newOptions.height = this.props.containerHeight.toString() + 'px';
-      newOptions.width = this.props.containerWidth.toString() + 'px';
     }
-    return (<Graph graph={this.props.graph} options={newOptions} events={this.props.events}/>)
+    return (
+      <div class={'net-graph-wrapper'}>
+        <Graph graph={this.props.graph} options={newOptions} events={this.props.events}/>
```

Comment:
Tried it like this?
```suggestion
        <Graph graph={this.props.graph} options={newOptions} events={this.props.events} className={'net-graph-wrapper'}/>
```

Reply:
@VakarisZ  Yeah, it doesn't work because the `Graph` component ignores the `className` property. https://github.com/crubier/react-graph-vis/blob/b86b4ebfe2112f3a6ae946ab3806c7d995502f3a/lib/index.js#L256-L260

I could add the style instead of adding a class name, but I'd rather we control as much of the style in `.css` files as possible, rather than having them hard coded in the JSX.